site.yml

```diff
@@ -1,6 +1,5 @@
 site:
   title: ownCloud Documentation
-  url: ${HOST}
```

Comment:
The default should be our production url

Reply:
yes, as I commented above

Reply:
What is the production URL?

Reply:
Currently it's https://owncloud-docs.netlify.com, in the end it will be https://doc.owncloud.com.

Reply:
Thanks. I'll change it to Netlify for now and then update it again later.